{"success": false, "error": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.8/dist-packages/mythril-0.23.15-py3.8.egg/mythril/interfaces/cli.py\", line 944, in parse_args_and_execute\n    execute_command(\n  File \"/usr/local/lib/python3.8/dist-packages/mythril-0.23.15-py3.8.egg/mythril/interfaces/cli.py\", line 838, in execute_command\n    report = analyzer.fire_lasers(\n  File \"/usr/local/lib/python3.8/dist-packages/mythril-0.23.15-py3.8.egg/mythril/mythril/mythril_analyzer.py\", line 172, in fire_lasers\n    issue.add_code_info(contract)\n  File \"/usr/local/lib/python3.8/dist-packages/mythril-0.23.15-py3.8.egg/mythril/analysis/report.py\", line 157, in add_code_info\n    codeinfo = contract.get_source_info(\n  File \"/usr/local/lib/python3.8/dist-packages/mythril-0.23.15-py3.8.egg/mythril/solidity/soliditycontract.py\", line 271, in get_source_info\n    file_index = mappings[index].solidity_file_idx\nIndexError: list index out of range\n", "issues": []}
